examples/numalogic-simple-pipeline/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM python:3.10-slim-bullseye AS builder
```

Comment:
is this right.. this will work only for M1 mac, not for everyone.. right?

Reply:
It works the other way round though. Let me try in an intel mac